tardis/io/logger/logger.py

```diff
@@ -149,6 +149,7 @@ def emit(self, record):
             stream_handler = logging.StreamHandler()
             stream_handler.setFormatter(logging.Formatter("%(name)s [%(levelname)s] %(message)s (%(filename)s:%(lineno)d)"))
             stream_handler.emit(record)
+            stream_handler.close()
```

Comment:
Does this mean the function opens and closes a StreamHandler for every log event?

Reply:
I'm sorry, I fixed it in https://github.com/tardis-sn/tardis/pull/3058/commits/3eb595c9fff6a83f7becc16289a4eb19045cc59d !